### PR TITLE
{system}[*/2015b] OpenSSL 1.0.1q (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

File: easybuild/easyconfigs/o/OpenSSL/OpenSSL-1.0.1p-foss-2015b.eb [hunk 1]
```diff
@@ -12,8 +12,6 @@ toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
 sources = [SOURCELOWER_TAR_GZ]
 source_urls = ['http://www.openssl.org/source/']
 
-patches = ['OpenSSL-%(version)s_icc-fixes.patch']
-
```
Comment (wpoely86) [inline easybuild/easyconfigs/o/OpenSSL/OpenSSL-1.0.1p-foss-2015b.eb]: why the sudden drop?

File: easybuild/easyconfigs/o/OpenSSL/OpenSSL-1.0.1q-foss-2015b.eb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ and Open Source toolchain implementing the Secure Sockets Layer (SSL v2/v3) and Transport Layer Security (TLS v1) 
+ protocols as well as a full-strength general purpose cryptography library. """
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.openssl.org/source/']
+
+patches = ['OpenSSL-%(version)s_icc-fixes.patch']
```
Comment (wpoely86) [inline easybuild/easyconfigs/o/OpenSSL/OpenSSL-1.0.1q-foss-2015b.eb]: in or out?

File: easybuild/easyconfigs/o/OpenSSL/OpenSSL-1.0.1p-foss-2015b.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/o/OpenSSL/OpenSSL-1.0.1p-foss-2015b.eb]: only needed for `icc`, which is not used in the `foss` toolchain, so shouldn't be there (don't fix it if it's not broken)

File: easybuild/easyconfigs/o/OpenSSL/OpenSSL-1.0.1q-intel-2015b.eb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+homepage = 'http://www.openssl.org/'
+description = """The OpenSSL Project is a collaborative effort to develop a robust, commercial-grade, full-featured,
+ and Open Source toolchain implementing the Secure Sockets Layer (SSL v2/v3) and Transport Layer Security (TLS v1) 
+ protocols as well as a full-strength general purpose cryptography library. """
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.openssl.org/source/']
```
Comment (wpoely86) [inline easybuild/easyconfigs/o/OpenSSL/OpenSSL-1.0.1q-intel-2015b.eb]: the `icc_fixes` is not need with actual `icc` :confused:

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/8ffe2748cfcd1a557b7a for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

File: easybuild/easyconfigs/o/OpenSSL/OpenSSL-1.0.1q-intel-2015b.eb (hunk 3 above)
Comment (boegel) [inline easybuild/easyconfigs/o/OpenSSL/OpenSSL-1.0.1q-intel-2015b.eb]: yes it is, see below ;)

**Pull request merged**